src/styles/notes.css

```diff
@@ -277,4 +280,58 @@
 
 .added-priority {
     background: radial-gradient(var(--white), var(--primary-color));
+}
+
+.pin-button {
+    background: none;
+    border: none;
+    margin-left: 20rem;
+    font-size: 1rem;
+    cursor: pointer;
+}
+
+.pinned {
+    color: var(--primary-color)
+}
+.unpinned {
+    color: var(--white)
+}
+.mg-top {
+    margin-top: 0.5rem;
+    position: absolute;
+    margin-left: 9rem;
+}
+.update-btn {
+    background-color: var(--primary-color);
+    color: var(--black);
+    font-size: 1.2rem;
+    font-weight: 700;
+    margin-left: -3rem;
+    margin-top: -0rem;
+    padding: 0.5rem;
+     border-radius: 20%;
+     cursor: pointer;
+}
+.date-div {
+   display: flex;
+   width: 20rem;
+   justify-content: space-between;
+   margin: auto;
+   font-weight: 500;
+}
+.cancel-btn-div {
+   position: absolute;
+   margin-top: -2.5rem;
+}
+.cancel-btn {
+   height: 2.5rem;
+   width: 2.5rem;
+   border-radius: 50%;
+   background-color: var(--navbar-color);
+   border: solid 2px var(--secondary-color);
+   cursor: pointer;
```

Comment:
Some CSS properties like cursor : pointer can be converted into a utility class. It could prevent repetition of code.

Reply:
Thanks for the suggestion, I will do it while refactoring the code later